.circleci/config.yml

```diff
@@ -8,6 +8,60 @@ jobs:
       - SDK_FILE: "openwrt-sdk-ar71xx-generic_gcc-7.3.0_musl.Linux-x86_64.tar.xz"
       - BRANCH: "master"
     steps:
+      - checkout:
+          path: ~/openwrt_packages
+
+      - run:
+          name: Check changes / verify commits
+          working_directory: ~/openwrt_packages
+          command: |
+             cat >> $BASH_ENV <<EOF
+             echo_red()   { printf "\033[1;31m\$*\033[m\n"; }
+             echo_green() { printf "\033[1;32m\$*\033[m\n"; }
+             echo_blue()  { printf "\033[1;34m\$*\033[m\n"; }
+             EOF
+             source $BASH_ENV
+
+             # Figure out commit_range ($COMMIT_RANGE returned)
+             .circleci/get_commit_range.sh
```

Comment:
This seems too complicated for me, I prefer to not realy on any CircleCI variables so it's easier to reproduce issue locally
I would prefer to use git commands only

Reply:
The reason for a separate shell script was for easy replacement. The complicated part deals with commit testing which we don't do yet. The simple case when looking at a PR is empirical. If you know the git commands to generate a COMMIT_RANGE from a PUSH, then mod this script. The CIRCLE_COMPARE_URL is empty on PRs if not mistaken.